app/features/shared/money-with-converted-amount.tsx

```diff
@@ -0,0 +1,41 @@
+import { MoneyDisplay } from '~/components/money-display';
+import { Skeleton } from '~/components/ui/skeleton';
+import { useExchangeRate } from '~/hooks/use-exchange-rate';
+import type { Money } from '~/lib/money';
+import { getDefaultUnit } from './currencies';
+
+export const MoneyWithConvertedAmount = ({ money }: { money: Money }) => {
+  const defaultFiatCurrency = 'USD';
+  const {
+    data: rate,
+    error: exchangeRateError,
+    isLoading: exchangeRateLoading,
+  } = useExchangeRate(
+    `${money.currency}-${money.currency === 'BTC' ? defaultFiatCurrency : 'BTC'}`,
```

Comment:
nit but `money.currency === 'BTC' ? defaultFiatCurrency : 'BTC'}` is repeated few times in this component. let's store it to some var and reuse:
```
const convertedCurrency = money.currency === 'BTC' ? defaultFiatCurrency : 'BTC';
```